ColorCoderTest.cpp

```diff
@@ -0,0 +1,34 @@
+#include "TeleCommColorCoder.h"
+
+
+void test_GetColorPairFromPairNumber(int pairNumber,
+    TeleCommunicationColorCoder::MajorColor expectedMajor,

```

Comment:
Try naming the test_ functions to include the expected outcome. Does that yield more test scenarios?